stochastic_physics_mpas.F

```diff
@@ -255,19 +279,36 @@ subroutine stochastic_physics_pattern_apply(domain, num_tends, tend_names, ierr)
       print*,'There are more than one block, in this MPI rank',  domain%dminfo%my_proc_id  
     endif
 
- ! get AR(1) pertubation_pattern
-
- ! retrieve the specified tendencies that needs to be perturbed
-   call mpas_pool_get_subpool(block%structs,'tend_physics' ,tend_physics)
+ ! retrieve the specified tendencies that needs to be perturbed and
+ ! apply the patterns (at all levels) to the tendency      
```

Comment:
So, the routine to apply the stochastic pattern to the tendencies is within stochastic_physics and not MPAS-model?  And the tendencies are passed to this routine, then passed back to MPAS?  Just want to confirm.